src/main/java/productDependency/CreateOrderPage.java

```diff
@@ -0,0 +1,1481 @@
+ 	 	public void enterQuantity10() throws IOException {
+ 	 		CreateOrderPage sp = new CreateOrderPage(driver);
+ 	 		log.info("Enter Quantity");
+ 	 		JavaScriptExec.sleep();
+ 	 		WaitClass.WaitForElementisDisplay(driver, 10, enterQuantity10);
+ 	 		Assert.assertTrue(enterQuantity10.isDisplayed());
+ 	 		enterQuantity10.clear();
+ 	 		enterQuantity10.sendKeys("4");
+ 	 		}
+ 	 	
+ 	 	
+ 	 	@FindBy(how = How.XPATH, using = "//*[@id='change--5.quantityAsDecimal']")
+ 	 //*[@id="change--5.quantityAsDecimal"]
+ 	 	private WebElement enterQuantity7;
+ 	 		/**
+ 	 		 * Method to Enter Price Rate.
+ 	 		 * 
+ 	 		 * @throws IOException
+ 	 		 */
+ 	 	public void enterQuantity7() throws IOException {
+ 	 		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+ 	 		log.info("Enter Quantity");
+ 	 		JavaScriptExec.sleep();
+ 	 		WaitClass.WaitForElementisDisplay(driver, 10, enterQuantity7);
+ 	 		Assert.assertTrue(enterQuantity7.isDisplayed());
+ 	 		enterQuantity7.clear();
+ 	 		enterQuantity7.sendKeys("3");
+ 	 		}
+	
+	
+	
+ 	 	@FindBy(how = How.XPATH, using = "//a//following::span[text()='Update'][2]")
+ 		private WebElement clickUpdateButton;
+
+ 		/**
+ 		 * Method to click on Update button.
+ 		 * 
+ 		 * @throws IOException
+ 		 */
+ 		public void clickUpdateButton() throws IOException {
+ 			CreateOrderPage sp = new CreateOrderPage(driver);
+ 			log.info("click on Update button.");
+ 			Assert.assertTrue(clickUpdateButton.isDisplayed());
+ 			clickUpdateButton.click();
+ 			JavaScriptExec.sleep();
+
+ 		
+
+	}
+	
+ 		@FindBy(how = How.XPATH, using = "//a//following::span[text()='Update'][2]")
+ 		private WebElement clickUpdateButton1;
+
+ 		/**
+ 		 * Method to click on Update button.
+ 		 * 
+ 		 * @throws IOException
+ 		 */
+ 		public void clickUpdateButton1() throws IOException {
+ 			CreateOrderPage sp = new CreateOrderPage(driver);
+ 			log.info("click on Update button.");
+ 			Assert.assertTrue(clickUpdateButton.isDisplayed());
+ 			clickUpdateButton.click();
+ 			JavaScriptExec.sleep();
+
+ 		
+
+	}
+	
+	@FindBy(how = How.XPATH, using = "//a[@class='submit save']//*[text()='Save Changes']")
+	private WebElement clickSaveChangesButton;
+
+	/**
+	 * Method to click on Save Changes Button.
+	 * 
+	 * @throws IOException
+	 */
+	public void clickSaveChangesButton() throws IOException {
+		OrderHierarchiesPage sp = new OrderHierarchiesPage(driver);
+		log.info("Click on Save Changes Button");
+		Assert.assertTrue(clickSaveChangesButton.isDisplayed());
+		JavaScriptExec.sleep();
+		clickSaveChangesButton.click();
+
+	}
+	
+	@FindBy(how = How.XPATH, using = "//*[@id='order-18257']/td[1]")
+	//*[@id="order-18257"]/td[1]
+	//*[@id="18258"]/span
+	private WebElement clickOrder;
+
+	/**
+	 * Method to click on Save Changes Button.
+	 * 
+	 * @throws IOException
+	 */
+	public void clickOrder() throws IOException {
+		OrderHierarchiesPage sp = new OrderHierarchiesPage(driver);
+		log.info("Click on Save Changes Button");
```

Comment:
@mayankchhajer Please rename the class name after pasting.

Reply:
Done